security-and-privacy-questionnaire-unrestricted-webusb.md

```diff
@@ -0,0 +1,125 @@
+
+(For Unrestricted WebUSB:) The Unrestricted WebUSB feature removes the restrictions on protected device classes so that trusted applications can access these devices.
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+Yes, an application with the `"usb-unrestricted"` permission receives the same information that is normally available through WebUSB API, except that the application additionally receives information about protected devices.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+(For WebUSB in general:) Yes, WebUSB exposes the serial number string descriptor if the device has one. The serial number is assumed to be unique among devices with the same vendor and product IDs and is therefore useful for fingerprinting. Specific devices may expose personal information related to the device capabilities.
+
+(For Unrestricted WebUSB:) Yes, the protected devices exposed by this feature are very likely to expose personal information including keystroke data, security credentials, webcam audio and video, and data stored on portable storage devices.
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+(For WebUSB:) The API is designed to be safe by default, exposing no information about connected devices until a user has granted permission. Additionally, the API blocks access to the most sensitive device classes. Typically these devices are accessible through another higher-level web platform API.
+
+(For Unrestricted WebUSB:) The feature does not attempt to protect sensitive information. We expect applications that use this feature will be carefully reviewed before publishing. If a published application is found to be abusing the permission then the application will be unpublished through a process external to this API.
+
+> 05.  Do the features in your specification introduce state
+>      that persists across browsing sessions?
+
+(For WebUSB in general:) Yes, permissions may persist across browsing sessions for devices that can be reliably re-identified by the implementation. In Chromium, this requires that the device has a non-empty serial number string descriptor. Permissions for devices without a serial number are revoked when the device is disconnected or the browser is closed.
+
+Additionally, a device may have its own state that persists independently of the browser.
+
+(For Unrestricted WebUSB:) No new information is persisted for this feature.
```

Comment:
Remove this line and the annotation since there's no additional state for Unrestricted WebUSB.

Reply:
Done